src/js/popup/view/timerCounterView.js

```diff
@@ -0,0 +1,20 @@
+import Timer from 'easytimer.js';
+
+const timer = new Timer();
+function updateTimerCounter(element, timer) {
+  element.innerText = timer.getTimeValues().toString();
+}
+
+export function startTimerCounter(initialSeconds) {
+  var options = initialSeconds
+    ? { precision: 'seconds', startValues: { seconds: initialSeconds } }
+    : null;
+  timer.start(options);
+
+  var element = document.getElementById("timer-counter");
```

Comment:
Can we move it to global area like const? This way we will get this element only once.